test/functionalTest/cases/1316_create_subscription/post_subs_errors.test

```diff
@@ -243,13 +243,13 @@ echo
 01. POST /v2/subscriptions (without id/idpattern)
 =================================================
 HTTP/1.1 400 Bad Request
-Content-Length: 84
+Content-Length: 93
 Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
 {
-    "description": "subject entities element has not id/idPattern",
+    "description": "subject entities element has not id/idPattern nor type",
```

Comment:
This is bad English (not from this PR, but from before). Why not change it now that were here ... ?

    subject entities element has no id/idPattern nor type